French Translation

AlphaGaming7780 opened the pull request:
Ping me 1 or 2 days before release, so I can push all things before, if you are OK.

Repost PR fix empty translation for 2 items

Laskyyy commented:
Suure thing, i'm likely getting some testing for today and then will release a small 2.0.4c patch later on monday or somethin, assuming I can make sure any of my recent changes haven't caused unintended side-effects ❤️

AlphaGaming7780 commented:
OK cool, if you can merge this PR before the update, that would be great,

Because I fixed 2 items that didn't have a French name and updated the translations following the modifications of the lang file en_us for create and create additions.

It would be great if I could get feedback on the translation from other French speakers.
